src/cli/doc/Cli.md

```diff
@@ -0,0 +1,56 @@
+# CLI
+
+This Dart application is a command line interface (CLI) designed to create Flutter projects based on a template.
+
+The repository comprises two main sections, the CLI (`src/cli`) and the Flutter application (`src/app`).
+
+The CLI will download the GitHub repository, and update the root content and the Flutter application when generating a new project, so nothing is bundled with the installation it's download-based and you will need internet.
```

Comment:
Prefer present tense over future tense. This could with `When generating a new project, the CLI downloads the GitHub repository to get the latest files. Because the template files are downloaded, you need internet when creating a new project.`